crtclient/docker/docker.go

```diff
@@ -373,3 +373,15 @@ func (d *Docker) GetContainerName(contID string) (string, error) {
 
 	return contInfo.Name, nil
 }
+
+// ExecContainer executes a specified in the container's namespace
+func (d *Docker) ExecContainer(contName string, cmdParams ...string) ([]byte, error) {
+	newCmdParams := append([]string{"exec", contName}, cmdParams...)
+	output, err := exec.Command("docker", newCmdParams...).CombinedOutput()
+	if err != nil {
+		log.Errorf("Error reading lldp information. Error: %s Output: \n%s\n", err, output)
```

Comment:
The error string should be more generic here and not refer lldp. Also no need for the extra `\n` at the end, the logger takes care of that.